build.gradle

```diff
@@ -168,6 +168,21 @@ task syncVersion {
     }
 }
 
+// Sync subproject versions with the version declared in the airship.properties file.
+task syncVersion
+syncVersion.dependsOn('syncRootVersion')
+syncVersion.dependsOn(':src:AirshipBindings.Android.ADM:syncVersion',
```

Comment:
I'm not sure enumerating this is necessary. When `syncVersion` is run at the top level, it should run all subproject `syncVersion`s.

Reply:
It wasn't doing that before I made this change.

Reply:
That is totally strange. I thought the whole idea with the subprojects was that they all run the tasks. Then all you have to do is specify them once, in settings.gradle. @rlepinski ?